python-sdk/tests/test_sdk.py

```diff
@@ -0,0 +1,210 @@
+import os
+import sys
+from typing import List, Union, Dict
+import pytest
+
+from sdk import RebuffSdk, RebuffDetectionResponse
+from utils import get_environment_variable
+
+try:
+    sys.path.insert(
+        0,
+        os.path.abspath(os.path.join(os.path.dirname(__file__), "../rebuff")),
+    )
+except NameError:
+    pass
+
+
+@pytest.fixture()
+def rebuff() -> RebuffSdk:
+    openai_apikey = get_environment_variable("OPENAI_API_KEY")
+    pinecone_apikey = get_environment_variable("PINECONE_API_KEY")
+    pinecone_index = get_environment_variable("PINECONE_INDEX_NAME")
+
+    rb = RebuffSdk(
+        openai_apikey,
+        pinecone_apikey,
+        pinecone_index,
+    )
```

Comment:
NIt: We could simplify this function a bit, and I don't think it looses readability:
```suggestion
    return RebuffSdk(
        get_environment_variable("OPENAI_API_KEY"),
        get_environment_variable("PINECONE_API_KEY"),
        get_environment_variable("PINECONE_INDEX_NAME"),
    )
```